script/seed.js

```diff
@@ -28,24 +30,27 @@ async function seed () {
   console.log(`seeded successfully`)
 }
 
-// Execute the `seed` function
-// `Async` functions always return a promise, so we can use `catch` to handle any errors
-// that might occur inside of `seed`
-seed()
+// Execute the `seed` function, IF we ran this module directly (`node seed`).
+// `Async` functions always return a promise, so we can use `catch` to handle
+// any errors that might occur inside of `seed`.
+if (module === require.main) {
+  seed()
   .catch(err => {
-    console.error(err.message)
-    console.error(err.stack)
+    console.error(err)
```

Comment:
I didn't see any good reason to run separate `console.error` calls for `err.message` and `err.stack`. Default `console.error(err)` logs both the message and stack trace already.